packages/core/src/transport/batch.ts

```diff
@@ -49,44 +27,35 @@ export class Batch {
     this.addOrUpdate(message, key)
   }
 
-  flush(flushReason: FlushReason, sendFn = this.request.send) {
-    if (this.bufferMessagesCount !== 0) {
-      const messages = this.pushOnlyBuffer.concat(objectValues(this.upsertBuffer))
-      const bytesCount = this.bufferBytesCount
-
-      this.flushObservable.notify({
-        bufferBytesCount: this.bufferBytesCount,
-        bufferMessagesCount: this.bufferMessagesCount,
-      })
+  flush(event: FlushEvent) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏it could be private now
```suggestion
  private flush(event: FlushEvent) {
```